src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+// #region header
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
```

Comment:
It seems like you have a redundant `text-align: center;` property in your `.header` class. Since you are using flexbox to align your items, this property has no effect and can be removed.